carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_carla_route

```diff
@@ -122,23 +120,25 @@ def initialize():
     
     #Ensure that localization is complete before checking route information
     localized_sub = rospy.Subscriber('/localization/current_pose', PoseStamped, pose_cb)
-    if local_pos.pose.position.x != 0 and local_pos.pose.position.y != 0:
-        if available_routes == None:
-            print("No routes available.")
-        else:
-            """Check if the selected route is one of the available routes. If it isn't,then print error statement"""
-            if check_selected_route(available_routes) == True:
-                route = selected_route
-                set_route(route.route_id)
-
+    while rospy.is_shutdown() != True:  
+        if local_pos.pose.position.x != 0 and local_pos.pose.position.y != 0:
```

Comment:
It is better if you could initialize the value for `local_pos.pose.position.x` and `local_pos.pose.position.y` as a infinite such as `float("inf")` because it might have a situation in both x and y of the vehicle position are 0. Or it is also better if you could just set a boolean value instead of actual position value since we just want to know is there any data flow into the topic.